docs/project/os-onboarding.md

```diff
@@ -0,0 +1,53 @@
+# OS Onboarding Guide
+
+Adding support for new operating systems (largely just new versions) is a frequent need. This guide describes how we do that, including policies we use.
+
+References:
+
+- [.NET OS Support Tracking](https://github.com/dotnet/core/issues/9638)
+- [Prereq container image lifecycle](https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/main/lifecycle.md)
+
+## Context
+
+In most cases, we find that new OSes _may_  uncover problems in dotnet/runtime and once resolved don't affect up-stack components or apps. This is because nearly all the APIs that touch native code (networking, cryptography) and deal with standard formats (time zones, ASN.1) are in dotnet/runtime. In many cases, we only see test breaks.

```

Comment:
```suggestion
In most cases, we find that new OS versions _may_  uncover problems in dotnet/runtime and once resolved don't affect up-stack components or apps. This is because nearly all the APIs that touch native code (networking, cryptography) and deal with standard formats (time zones, ASN.1) are in dotnet/runtime. In many cases, we only see test breaks.

```